<launch>
  <node pkg="inertiallabs_ins" type="il_ins" name="ins" output="screen">
   <remap from="~/gnss_fix_fused" to="/hardware_interface/gnss_fix_fused"/>
   <remap from="~/imu_raw" to="/hardware_interface/imu/imu_raw"/>
   <remap from="~/discovery" to="/hardware_interface/driver_discovery"/>
   <param name="ins_url" type="string" value="serial:/dev/ttyUSB0:115200" />
   <param name="ins_output_format" type="int" value="87"/>
  </node>
</launch>
